### PR TITLE
HOWTO: connect service with Elastic stack

### PR DESCRIPTION
**Pull request by mtojek**
Issue: https://github.com/elastic/elastic-package/issues/190

Preview: https://github.com/mtojek/elastic-package/blob/docs-run-service/docs/howto/connect_service_with_elastic_stack.md

### CODE REVIEW

File: docs/howto/connect_service_with_elastic_stack.md [hunk 1]
```diff
@@ -0,0 +1,73 @@
...
+stack instead of linking it temporarily for the execution of system tests. It's especially useful if you're adjusting Kibana
+dashboards or want to examine metrics data collected for the service in the Elasticsearch index.
+
+The procedure assumes an existence of two Docker networks - stack network and service network. The goal is to expose
+the stack network to the service container. The Elastic Agent will have an access to the service metrics endpoint and its
+logs (via mounted volume).
+
+## User guide (based on the Apache integration)
+
+1. If you haven't done it already, please boot up the Elastic stack:
```
Comment (ChrsMark) [inline docs/howto/connect_service_with_elastic_stack.md:15]: Shouldn't the note about logs volume mounting be mentioned early on?
That you need to add the volume mount part in agent service so as to have logs to be parsed by agent?
```yaml
volumes:
  - ${SERVICE_LOGS_DIR}:/usr/local/apache2/logs
```
```

File: docs/howto/connect_service_with_elastic_stack.md [hunk 2]
```diff
@@ -0,0 +1,73 @@
...
+
+```bash
+elastic-package stack up -d -v
```
Comment (ChrsMark) [inline docs/howto/connect_service_with_elastic_stack.md:18]: Also let's make it more clear that the compose file is located at `~/.elastic-package/stack/snapshot.yml ` for those that are not familiar with the internals.

**Review by ChrsMark — APPROVED** [review]
lgtm! Went over it live while testing manually my stuff and it can cover the manual testing process. Left some comments about some fuzzy points that i was able to by pass but would be better to clarify them more. 

Let me know what you think!

File: docs/howto/connect_service_with_elastic_stack.md [hunk 3]
```diff
@@ -0,0 +1,73 @@
...
+Server: Apache/2.4.20 (Unix)
+Content-Type: text/html; charset=ISO-8859-1
+```
+
+Check presence of application log files:
+
+```bash
+ls /tmp/service_logs
+access.log  error.log  httpd.pid
+```
```
Comment (ChrsMark) [inline docs/howto/connect_service_with_elastic_stack.md:73]: Also add a last but not least comment that the integration should point to the targeted service with the service name like: `elastic-package-service_apache_1:80` to make it easier for testers to set the integration from within the Fleet UI.

Screenshot 2020-11-26 at 17 26 46

File: docs/howto/connect_service_with_elastic_stack.md (hunk 1 above)
Comment (mtojek) [reply docs/howto/connect_service_with_elastic_stack.md]: I put an additional section describing the service Docker Compose files. Regarding the Elastic Agent definition, the volume property is already there:

https://github.com/elastic/elastic-package/blob/master/internal/install/static_snapshot_yml.go#L96

File: docs/howto/connect_service_with_elastic_stack.md (hunk 2 above)
Comment (mtojek) [reply docs/howto/connect_service_with_elastic_stack.md]: I added the "Advanced" section to cover it, but in general this method assumes that you don't interact with `snapshot.yml`, but can use `docker` command to achieve the goal.

File: docs/howto/connect_service_with_elastic_stack.md (hunk 3 above)
Comment (mtojek) [reply docs/howto/connect_service_with_elastic_stack.md]: Good catch! I mentioned the point about the service address.

**Comment by mtojek** [comment]
@ChrsMark would you mind re-reviewing this PR?

**Review by ChrsMark — APPROVED** [review]
LGTM! thanks!

File: docs/howto/connect_service_with_elastic_stack.md (hunk 1 above)
Comment (ChrsMark) [reply docs/howto/connect_service_with_elastic_stack.md]: nice!

**Pull request merged**